pkg/webhooks/sdnmigration/sdnmigration.go

```diff
@@ -46,6 +47,10 @@ type NetworkConfigWebhook struct {
 // Authorized will determine if the request is allowed
 func (w *NetworkConfigWebhook) Authorized(request admissionctl.Request) admissionctl.Response {
 	// allow if modified by an allow listed service account
+	if priviledgeUserAccount == request.UserInfo.Username {
+		return utils.WebhookResponse(request, true, "Priviledge user may access")
```

Comment:
Typo: `Privileged user may access` to follow the existing pattern.

Reply:
Change done